controllers/rhmi/bootstrapReconciler.go

```diff
@@ -179,45 +165,37 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
+	if observabilityConfig.GetNamespace() == "" {
+		observabilityConfig.SetNamespace(ns)
+		err := r.ConfigManager.WriteConfig(observabilityConfig)
 		if err != nil {
 			return rhmiv1alpha1.PhaseFailed, err
 		}
-		ns := observability.GetDefaultNamespace(r.installation.Spec.NamespacePrefix)
-		if observabilityConfig.GetNamespace() == "" {
-			observabilityConfig.SetNamespace(ns)
-			err := r.ConfigManager.WriteConfig(observabilityConfig)
-			if err != nil {
-				return rhmiv1alpha1.PhaseFailed, err
-			}
-		}
-		phase, err = r.ReconcileNamespace(ctx, observabilityConfig.GetNamespace(), installation, serverClient, log)
-		if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
-			events.HandleError(r.recorder, installation, phase, "Failed to create observability operand namespace", err)
-			return phase, errors.Wrap(err, "failed to create observability operand namespace")
-		}
+	}
+	phase, err = r.ReconcileNamespace(ctx, observabilityConfig.GetNamespace(), installation, serverClient, log)
+	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+		events.HandleError(r.recorder, installation, phase, "Failed to create observability operand namespace", err)
+		return phase, errors.Wrap(err, "failed to create observability operand namespace")
 	}
 
-	// temp code for rhmi 2.8 to 2.9.0 upgrades, remove this when all clusters upgraded to 2.9.0
-	r.deleteObsoleteService(ctx, serverClient)
-
-	if integreatlyv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
-		if err = r.processQuota(installation, request.Namespace, installationQuota, serverClient); err != nil {
-			events.HandleError(r.recorder, installation, integreatlyv1alpha1.PhaseFailed, "Error while processing the Quota", err)
-			installation.Status.LastError = err.Error()
-			return integreatlyv1alpha1.PhaseFailed, err
-		}
-		metrics.SetQuota(installation.Status.Quota, installation.Status.ToQuota)
-
-		// temp code for RHOAM, remove once all clusters are upgraded to 1.14
-		// Remove all prometheus rules under redhat/sandbox-rhoam/rhoami-operator
-		phase, err = r.removePrometheusRules(ctx, serverClient, installation.Spec.NamespacePrefix)
-		if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
-			events.HandleError(r.recorder, installation, phase, "Failed to remove existing prometheus rules from rhoam-operator namespace", err)
-			return phase, errors.Wrap(err, "Failed to remove existing prometheus rules from rhoam-operator namespace")
-		}
+	if err = r.processQuota(installation, request.Namespace, installationQuota, serverClient); err != nil {
+		events.HandleError(r.recorder, installation, integreatlyv1alpha1.PhaseFailed, "Error while processing the Quota", err)
+		installation.Status.LastError = err.Error()
+		return integreatlyv1alpha1.PhaseFailed, err
+	}
+	metrics.SetQuota(installation.Status.Quota, installation.Status.ToQuota)
 
+	// temp code for RHOAM, remove once all clusters are upgraded to 1.14
```

Comment:
We don't need this code anymore.

Reply:
This is code related to RHOAM not RHMI. There is enough change happening with the RHMI removal. The removal of this is out side the scope of this work.

Reply:
As a general rule that we have agreed on during one of our sessions, it is recommended to make code improvements when we can while working on other jiras. There is no harm in deleting a couple of lines that are not needed. Let's tidy up our code since we are looking line by line!

Reply:
Thanks